src/main/java/co/inventorsoft/academy/service/FileWriterService.java

```diff
@@ -15,8 +14,7 @@ public class FileWriterService {
      */
     public void write(String str) {
         try {
-            BufferedWriter writer = new BufferedWriter(
-                    new java.io.FileWriter("data.csv", true));
+            BufferedWriter writer = new BufferedWriter(new java.io.FileWriter("data.csv", true));
             writer.write(str + "\n");
             writer.close();
         } catch (IOException e) {
```

Comment:
Printing a stacktrace is bad experience. Try to use some logger (for example logback or slf4j)